cmd/admission-webhook-server/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+
+	controller "github.com/kinvolk/lokomotive/internal/admission-webhook-server"
+)
+
+const (
+	port = "8080"
+)
+
+func usage() {
+	flag.PrintDefaults()
+	os.Exit(0)
+}
+
+func returnError(msg string, err error) {
+	glog.Fatalf("%s: %v", msg, err)
+}
+
+func main() {
+	var tlscert, tlskey, logtostderr, stderrthreshold, loglevel string
+
+	flag.Usage = usage
+
+	flag.StringVar(&tlscert, "tlsCertFile", "/etc/certs/cert.pem", "File containing the x509 Certificate for HTTPS.")
+	flag.StringVar(&tlskey, "tlsKeyFile", "/etc/certs/key.pem", "File containing the x509 private key to --tlsCertFile.")
+	flag.StringVar(&logtostderr, "logToStderr", "true", "Logs are written to files instead of to standard error.")
+	flag.StringVar(&stderrthreshold, "stdErrThreshold", "WARNING", "Severity level for standard error.")
+	flag.StringVar(&loglevel, "loglevel", "2", "Enable V-leveled logging at the specified level.")
+	flag.Parse()
+
+	err := flag.Set("logtostderr", logtostderr)
```

Comment:
What uses this flag? I don't understand what this code does. If we set this, I assume something reads it? But if you say that the flag is already defined, are we trying to override the defaults? Do we need to do it or maybe we can just set them on the deployment level?

Reply:
They are required for glog to write logs to file, and yes they can be done at deployment level. Doing that instead of setting them here